src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx

```diff
@@ -162,6 +173,7 @@ const EnrolmentTable: React.FC<Props> = ({
               'occurrenceDetails.enrolmentTable.showEnrolmentDetails'
             )}
             {...row.getToggleRowExpandedProps()}
+            aria-expanded={row.isExpanded ? true : false}
```

Comment:
isExpanded already sounds like a boolean value.

Reply:
yeah, for some reason isExpanded is undefined when closed, so had to do this

Reply:
Ok, makes sense

Reply:
But maybe comment would be nice here